v4/source/posts.yaml

```
@@ -417,6 +417,48 @@
         '403':
           $ref: '#/responses/Forbidden'
 
+  '/users/{user_id}/channels/{channel_id}/posts/unread':
+    get:
+      tags:
+        - posts
+      summary: Get posts around last unread
+      description: |
+        Get posts around last unread post by a user in a channel. Use the query parameters `limit_before` and `limit_after` to modify the number of posts returned from this endpoint.
```

Just a suggestion to make the API's purpose a bit clearer. Also, I don't think we need to describe the parameters up here since they're explained below
```suggestion
        Get the oldest unread post in the channel for the given user as well as the posts around it.
```

---

Thanks, applied the suggestion.